mindmaps-dashboard/src/js/visualiser/Visualiser.js

```diff
@@ -50,7 +50,17 @@ export default class Visualiser {
         this.networkConfig = {
             autoResize: true,
             edges: { arrows: { to: true } },
-            physics: { solver: "forceAtlas2Based" },
+            physics: {
```

Comment:
This is one of the reasons I hate JS. These dynamically typed embedded objects are so painful to work with. One day the library will render one of these parameters useless and you would never know.